python/importer/remove_old_data.py

```diff
@@ -0,0 +1,22 @@
+import psycopg2
+import logging
+from common.utils import get_conn_params
+
+def main():
+    logger = logging.getLogger('importer')
+
+    conn = psycopg2.connect(**get_conn_params())
+    try:
+        with conn:
+            with conn.cursor() as cur:
+                logger.info("Removing data.")
+                # Remove data that is older than 2 weeks
+                cur.execute("DELETE FROM hfp.observed_journey WHERE oday < now() - interval '2 week'")
+                cur.execute("DELETE FROM observation WHERE oday < now() - interval '2 week'")
+                # Not all hfp_points will get deleted due to ON DELETE CASCADE rule, so we have to clean up the rest (that are not related to observed_journeys) with:
+                cur.execute("DELETE FROM hfp.hfp_point WHERE point_timestamp < now() - interval '2 week'")
+    finally:
```

Comment:
At least a `except psycopg2.OperationalError as e:` block with `logger.error(...)` would be good here, so common errors such as db connection error or typos in SQL statements are not passed silently by the `finally` block.